### PR TITLE
Add clone command example

### PR DESCRIPTION
**Pull request by dacharyc**

### CODE REVIEW

File: content/setting-up-services/how-to-set-up-services/leverage-service-commands.md [hunk 1]
```diff
@@ -128,6 +128,20 @@ commands that you run in `clone` get committed as part of the build snapshot, so
 Preview. Things like [Reset](/building-a-preview/administer-previews/change-preview-states/#reset) do not erase `clone`
 commands, and those commands would carry over if you made a Preview cloned in this way a Base Preview.
 
+You might use the `clone` command in cases where your running Previews depend on something written to disk that is
+derived from the [Preview ID or Service URL](/faq/find-tugboat-ids/); or anything that will have new values in the
```
Comment (bchavet) [inline content/setting-up-services/how-to-set-up-services/leverage-service-commands.md:132]:
```suggestion
derived from the [Preview ID](/faq/find-tugboat-ids/) or [Service URL](reference/environment-variables/); or anything that will have new values in the
```

**Comment by dacharyc** [comment]
Sorry, @bchavet , I noticed this before the weekend but then forgot about it by Monday. Do you want to give it another review, and then update the branch and merge if you're happy with it?

**Comment by bchavet** [comment]
Test this please

File: content/setting-up-services/how-to-set-up-services/leverage-service-commands.md [hunk 2]
```diff
@@ -128,6 +128,20 @@ commands that you run in `clone` get committed as part of the build snapshot, so
 Preview. Things like [Reset](/building-a-preview/administer-previews/change-preview-states/#reset) do not erase `clone`
 commands, and those commands would carry over if you made a Preview cloned in this way a Base Preview.
 
+You might use the `clone` command in cases where your running Previews depend on something written to disk that is
+derived from the [Preview ID](/faq/find-tugboat-ids/) or [Service URL](reference/environment-variables/); or anything that will have new values in the
```
Comment (bchavet) [inline content/setting-up-services/how-to-set-up-services/leverage-service-commands.md:132]: Screen Shot 2020-07-30 at 9 57 33 AM

```suggestion
derived from the [Preview ID](/faq/find-tugboat-ids/) or [Service URL](/reference/environment-variables/); or anything that will have new values in the
```

**Pull request merged**